payment_paypal/indico_payment_paypal/plugin.py

```diff
@@ -31,6 +34,12 @@ class EventSettingsForm(PaymentEventSettingsFormBase):
     business = StringField(_('Business'), [UsedIf(lambda form, _: form.enabled.data), DataRequired(),
                                            validate_business],
                            description=_('The PayPal ID or email address associated with a PayPal account.'))
+    paypal_fixed_fee = StringField(_('PayPal Fixed Transaction Fee'), [UsedIf(lambda form, _: form.enabled.data), DataRequired(),
+                                           validate_paypal_fixed_fee],
+                           description=_('The PayPal transaction fixed fee.'))
+    paypal_percent_fee = StringField(_('PayPal Percentage Transaction Fee'), [UsedIf(lambda form, _: form.enabled.data), DataRequired(),
+                                           validate_paypal_percent_fee],
+                           description=_('The PayPal transaction percentage fee.'))
```

Comment:
it doesn't make much sense to use string fields for this, WTForms has a `DecimalField` that would be ideal for these values, and if you use the validator `NumberRange(min=0, max=999999999.99)` it does all of the validation without the need for regex

Reply:
done